vars/jobsParameters.groovy

```diff
@@ -152,7 +152,7 @@ for (image in res) {
    result.add(image.getImageIds());
 }
 
-return result[0].imageTag.sort().reverse().findAll{it.startsWith(rancher_cluster_name.trim() + '-' + rancher_project_name.trim())};
+return result[0].imageTag.sort().reverse().findAll().findAll{it.startsWith(rancher_cluster_name.trim() + '-' + rancher_project_name.trim())};
```

Comment:
Is it needed here findAll two times? Or it is just typo

Reply:
Yep, we need this one empty .findAll() cause it's cleaning list from null values(untagged images) in ECR.